src/containers/sidebar/SidebarContainer.ts

```diff
@@ -37,21 +42,13 @@ export const SidebarContainer = createContainer<SidebarValue>(() => {
     }
 
     const value = input.trim();
-
-    if (value !== '') {
-      worker.requestFilter({
-        input,
-        newItems,
-        passedItems,
-        failedItems,
-        deletedItems,
-      });
-    } else {
-      setNewItems(toStructualItems(entities.newItems));
-      setPassedItems(toStructualItems(entities.passedItems));
-      setFailedItems(toStructualItems(entities.failedItems));
-      setDeletedItems(toStructualItems(entities.deletedItems));
-    }
```

Comment:
This block cause this issue.
When remove words rapidly, set all items in else block for now.
However it is overwritten by debounced worker response.